[2025-07-12T15:15:19.993+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flight_ml_dag.detect_flight_anomalies manual__2025-07-12T15:15:18.096672+00:00 [queued]>
[2025-07-12T15:15:19.996+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flight_ml_dag.detect_flight_anomalies manual__2025-07-12T15:15:18.096672+00:00 [queued]>
[2025-07-12T15:15:19.997+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-07-12T15:15:20.002+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): detect_flight_anomalies> on 2025-07-12 15:15:18.096672+00:00
[2025-07-12T15:15:20.005+0000] {standard_task_runner.py:60} INFO - Started process 348 to run task
[2025-07-12T15:15:20.007+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'flight_ml_dag', 'detect_flight_anomalies', 'manual__2025-07-12T15:15:18.096672+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/flight_ml_dag.py', '--cfg-path', '/tmp/tmp5vait3ht']
[2025-07-12T15:15:20.009+0000] {standard_task_runner.py:88} INFO - Job 85: Subtask detect_flight_anomalies
[2025-07-12T15:15:20.044+0000] {task_command.py:423} INFO - Running <TaskInstance: flight_ml_dag.detect_flight_anomalies manual__2025-07-12T15:15:18.096672+00:00 [running]> on host f1b3ce86e5d1
[2025-07-12T15:15:20.075+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_scientist' AIRFLOW_CTX_DAG_ID='flight_ml_dag' AIRFLOW_CTX_TASK_ID='detect_flight_anomalies' AIRFLOW_CTX_EXECUTION_DATE='2025-07-12T15:15:18.096672+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-12T15:15:18.096672+00:00'
[2025-07-12T15:15:20.079+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-07-12T15:15:20.108+0000] {logging_mixin.py:188} INFO - ‚úÖ Anomaly detection completed
[2025-07-12T15:15:20.108+0000] {logging_mixin.py:188} INFO - üö® Total anomalies found: 99
[2025-07-12T15:15:20.108+0000] {logging_mixin.py:188} INFO - ‚è±Ô∏è Duration anomalies: 5
[2025-07-12T15:15:20.108+0000] {logging_mixin.py:188} INFO - üìÖ Scheduling anomalies: 86
[2025-07-12T15:15:20.109+0000] {logging_mixin.py:188} INFO - üìä Performance anomalies: 8
[2025-07-12T15:15:20.109+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-12T15:15:20.113+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=flight_ml_dag, task_id=detect_flight_anomalies, execution_date=20250712T151518, start_date=20250712T151519, end_date=20250712T151520
[2025-07-12T15:15:20.148+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-12T15:15:20.157+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
